src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -674,7 +674,7 @@ internal void ResetEngine(EngineController controller, bool markNodesAsDirty = f
                 MarkNodesAsModifiedAndRequestRun(Nodes);
             }
 
-            if (RunSettings.RunEnabled && RunSettings.RunType == RunType.Automatic && forceAutomaticWithoutRun == false)
+            if (RunSettings.RunEnabled && RunSettings.RunType == RunType.Automatic && RunSettings.ForceAutomaticWithoutRun == false)
```

Comment:
Can also be written as `if (RunSettings.RunEnabled && RunSettings.RunType == RunType.Automatic && !RunSettings.ForceAutomaticWithoutRun)`